src/AssetTransferApi.ts

```diff
@@ -306,12 +306,13 @@ export class AssetTransferApi {
 
 		let txMethod: Methods;
 		let transaction: SubmittableExtrinsic<'promise', ISubmittableResult>;
-
 		if (
 			(xcmPallet === XcmPalletName.xTokens || xcmPallet === XcmPalletName.xtokens) &&
-			(xcmDirection === Direction.ParaToSystem || xcmDirection === Direction.ParaToPara)
+			(xcmDirection === Direction.ParaToSystem ||
+				xcmDirection === Direction.ParaToPara ||
+				xcmDirection === Direction.ParaToRelay)
 		) {
-			if (!paysWithFeeDest && assetIds.length < 2) {
+			if (xcmDirection === Direction.ParaToRelay || (!paysWithFeeDest && assetIds.length < 2)) {
```

Comment:
Since we check that `xcmDirection` is possibly `ParaToRelay` on line `313` is the additional `xcmDirection === Direction.ParaToRelay` equality check needed?

Reply:
So the reason I did this was to enforce that the TX was always `transferMultiAsset`.

Just incase so if a fee portion was added it will still resolve to `transferMultiAsset`.

Reply:
nice! that makes sense

Reply:
I'll write a comment there to clarify though!